cu29/src/curuntime.rs

```diff
@@ -120,24 +120,47 @@ pub struct CuExecutionStep {
     pub culist_output_index: Option<u32>,
 }
 
-fn find_output_index_from_nodeid(node_id: NodeId, steps: &Vec<CuExecutionStep>) -> Option<u32> {
+/// This structure represents a loop in the execution plan.
+/// It is used to represent a sequence of steps that are executed multiple times.
```

Comment:
It's not as simple as a sequence of steps since it represents either sequences of steps or sequences _of_ sequences of steps, right?

Reply:
yes, fixed.